lms/templates/view.html

```diff
@@ -97,7 +100,16 @@ <h5 class="test-name">
     </div>
     {% if is_manager and not shared_url %}
     <div id="popular-comments">
-      <div id="exercise-comments">
+      <div id="exercise-grade" class="{{ direction }}">
+        <h2>{{ _('ציון התרגיל') }}</h2>
+        <div class="btn-group-vertical" id="solution-grade" role="group" aria-label="grades radio toggle button group">
+          {% for grade_mark in grade_marks %}
+            <input type="radio" class="btn-check" name="grade-mark" id="btnradio{{ loop.index }}" autocomplete="off" value="{{ grade_mark.id }}" {%- if solution['grade_mark'] and solution['grade_mark']['name'] == grade_mark.name %}checked{% else %}{% if loop.index == 1 %}checked{% endif %}{% endif -%}>
```

Comment:
`solution.grade_mark` (stick to the dot notation if possible)

Reply:
Also, I think the `.` notation is error free, so you can write `solution.grade_mark.name == grade_mark.name or loop.first`